freqtrade/exchange/exchange.py

```diff
@@ -1964,6 +1964,37 @@ def get_funding_fees(self, pair: str, amount: float, open_date: datetime) -> flo
         else:
             return 0.0
 
+    @retrier
+    def get_liquidation_price(self, pair: str):
+        '''
+            Set's the margin mode on the exchange to cross or isolated for a specific pair
+            :param pair: base/quote currency pair (e.g. "ADA/USDT")
+        '''
+        if self._config['dry_run'] or not self.exchange_has("fetchPositions"):
+            # Some exchanges only support one collateral type
```

Comment:
is this comment really related to this if branch?

i don't think that having one or multiple margin types will have an impact on the availability of fetchPositions ... ?